### DISCUSSION TITLE
How to draw out of canvas?

### DISCUSSION THREAD

**Discussion by amoraleite** [keiwando/toonsquid#508] [Q&A]
Guys I'm trying to draw a scenery of my animation, this scenery will move behind of the character, but I can't draw out of canvas area... in the ToonSquid I just can draw into the canvas area? There is any workaround?

**Comment by amoraleite** [comment]
> Guys I'm trying to draw a scenery of my animation, this scenery will move behind of the character, but I can't draw out of canvas area... in the ToonSquid I just can draw into the canvas area? There is any workaround?

I figured out!
Workaround:
Vector can be drawing out of the canvas...

**Comment by kavlor** [comment]
You can also create in a higher resolution project then export animation as a project or animation file (MP4,MOV) then import into your library of your lower resolution project. 4096*4096 seems to be the max res.

If you want to further edit the background you can use edit clip and you will be in the higher resolution , when you go back to the main scene it will update.

**Reply by Cat-Butler** [reply]
If the background is static, exporting a PNG file will also work for @kavlor's solution, or even multiple PNG files with transparency for parallax effects.

Another option is to add a Camera layer in the timeline, which will pick up the current project canvas size.
The new Crop feature introduced in 1.10, in spite of its name, then allows you to increase the canvas size e.g. doubling the width, in the same project. All layers, including pixel paint layers, will pick up the new expanded size, so you can paint "outside the camera view" although not, strictly speaking, "outside the canvas".
You can then either move an extra-width pixel layer or layers across the old-size camera view, or move the camera along the new extra-width canvas.
